rpcs3/Emu/RSX/VK/vkutils/sync.cpp

```diff
@@ -116,6 +116,24 @@ namespace vk
 		}
 	}
 
+	void event::host_signal() const
+	{
+		ensure(m_vk_event);
+		vkSetEvent(m_device, m_vk_event);
+	}
+
+	void event::reset() const
+	{
+		if (m_vk_event) [[likely]]
+		{
+			vkResetEvent(m_device, m_vk_event);
+		}
+		else
+		{
+			*m_value = 0xCAFEBABE;
```

Comment:
no nullptr check?

Reply:
No. It cannot be null, though an ensure can be added if needed.
This is device memory, if we couldn't allocate it we would have had a vulkan assert.